_posts/2018-09-17-pulp-community-health-audit.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Pulp Community Health Audit Report
+author: Brian Bouterse
+---
+In August 2018 the [Open Source and Standards Group](http://community.redhat.com/) completed their
+first Pulp Community Health Audit. This annual audit is a chance to assess the health and diversity
+of the Pulp community, identify growth areas, apply best practices, and check-in on progress made
+towards previous goals.
+
+This blog gives a summary, but you can read the full report
+[here](/files/Community_Audit_Report_Pulp_2018.pdf).
+
+
+# Overview
+
+Overall the Pulp community is healthy, but it
+can improve by making most of its resources more clearly discoverable, e.g. quick start guides, etc.
+Here are the recommended actions for the Pulp community to do to improve itself from the audit:
+
+* Create a quick start documentation guide
+* Create a more visible link to the PUPs
+* Create an online calendar to create a steady-state place for community members to discover events
+* Create a direct link to release notes on the Pulp web site
+* Review PUPs to see if a Tech Advisory Board or other adjustments need to be made to governance
+  model
+* Explore if Pulp would benefit from a part-time or full-time outreach and marketing person
+
+
+# Infrastructure and Process
+
+Recommendations to Pulp come in two areas here: Resource Discoverability and Process Improvements.
+
+Pulp needs to focus on its website and docs some to ensure users can find what they need quickly.
+For example, this comes through more quickstart guides, and fewer clicks required to get users to
```

Comment:
Some suggested improvements include more quickstart guides and fewer page clicks for users to get to key pages such as the installation page.

Reply:
ty